### PR TITLE
Feature/zen 15800 ceilometer

### PR DESCRIPTION
**Pull request by pcarinhas**
This partially fixes ZEN-15800
NOTE: Don't Merge untill John C. has reviewed..

### CODE REVIEW

File: ZenPacks/zenoss/PythonCollector/service_definition/service.json [hunk 1]
```diff
@@ -0,0 +1,387 @@
+{
+   "servicePath": "/hub/collector"
+   "ServiceDefinition": {
+       "Description": "Custom: Adds a \"Python\" data source type. Adds RabbitMQ imports",
```
Comment (cluther) [inline ZenPacks/zenoss/PythonCollector/service_definition/service.json]: The user will see this description in Control Center. The "Custom: " prefix and RabbitMQ import detail are probably irrelevant.

File: ZenPacks/zenoss/PythonCollector/service_definition/service.json [hunk 2]
```diff
@@ -0,0 +1,387 @@
+{
+   "servicePath": "/hub/collector"
```
Comment (jafcrocker) [inline ZenPacks/zenoss/PythonCollector/service_definition/service.json]: Needs a comma here...

**Comment by jafcrocker** [comment]
Diffs:
```
jcrocker@jcrocker-dev:~/tmp/ZenPacks.zenoss.PythonCollector$ diff <(jq .ServiceDefinition ZenPacks/zenoss/PythonCollector/service_definition/service.json) <(jq . $(zendev root)/src/service/services/Zenoss.resmgr/localhost/localhost/zenpython/service.json)
70,74d69
<     },
<     {
<       "Purpose": "import",
<       "Application": "rabbitmq.*",
<       "Name": "rabbitmq"
95c90
<   "Description": "Adds a \"zenpython\" data source type for PythonCollector",
---
>   "Description": "Adds a \"Python\" data source type for asynchronous collection using a custom Python module",
```

**Comment by pcarinhas** [comment]
I've tested this against zendev, by uninstalling/installing, and rabbit now listens on zenpython.
You must test with telnet, as no port shows in netstat. Must be iptables DNAT thing.. Other ports do show in netstat.

**Pull request merged**